### PR TITLE
U-Boot and RaspberryPi 4

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

File: recipes-kernel/linux/linux-raspberrypi_4.19.bb [hunk 1]
```diff
@@ -3,7 +3,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi:"
 LINUX_VERSION ?= "4.19.57"
 LINUX_RPI_BRANCH ?= "rpi-4.19.y"
 
-SRCREV = "55b89745a4782db50e86291bd1adee8fb44b84ad"
+SRCREV = "e95c533eba52d1879d94bdd1ede00548ad1cf622"
```
Comment (kraj) [inline recipes-kernel/linux/linux-raspberrypi_4.19.bb]: Can you bump it to https://github.com/raspberrypi/linux/commit/246113692edbef9a438b31ab2dd0172a30ed5eb2 and also change `LINUX_VERSION to 4.19.58`

File: conf/machine/raspberrypi4-64.conf [hunk 2]
```diff
@@ -19,10 +19,14 @@ RPI_KERNEL_DEVICETREE = " \
 SDIMG_KERNELIMAGE ?= "kernel8.img"
 SERIAL_CONSOLES ?= "115200;ttyS0"
 
-MACHINE_FEATURES_append = " armstub vc4graphics"
-VC4DTBO ?= "vc4-fkms-v3d"
```
Comment (kraj) [inline conf/machine/raspberrypi4-64.conf]: Dont we need `fkms` still for 64bit

File: conf/machine/raspberrypi4.conf [hunk 3]
```diff
@@ -11,7 +11,8 @@ MACHINE_EXTRA_RRECOMMENDS += "\
     bluez-firmware-rpidistro-bcm4345c0-hcd \
 "
 
-SDIMG_KERNELIMAGE ?= "kernel7.img"
+# 'l' stands for LPAE
+SDIMG_KERNELIMAGE ?= "kernel7l.img"
```
Comment (kraj) [inline conf/machine/raspberrypi4.conf]: can we use LPAE for older rpi as well ?

Comment (agherzan) [reply conf/machine/raspberrypi4.conf]: Most probably for RPi3 yes. But I'll let that testing be part of another PR.

File: conf/machine/raspberrypi4-64.conf (hunk 2 above)
Comment (agherzan) [reply conf/machine/raspberrypi4-64.conf]: We do. That slipped in by mistake. Fixed.

File: recipes-kernel/linux/linux-raspberrypi_4.19.bb (hunk 1 above)
Comment (agherzan) [reply recipes-kernel/linux/linux-raspberrypi_4.19.bb]: Agreed. Pushed.

**Pull request merged**